core/templates/services/rte-helper.service.spec.ts

```diff
@@ -38,6 +38,7 @@ describe('Rte Helper Service', () => {
         schema: {
           type: 'unicode'
         },
+        default_value_obtainable_from_highlight: false,
```

Comment:
Please add tests for the changes you introduce.

Reply:
@nikitaevg Is there any shared file for RTE unit tests or e2e tests?

Reply:
Try [this](https://github.com/oppia/oppia/blob/develop/core/tests/webdriverio_desktop/extensions.js), but I'm not sure how difficult it would be.

@seanlip do you know a better place for RTE tests?

Reply:
Generally each RTE component has its own e2e helper functions in the relevant folder in extensions/rich_text_components. These tests are written in a generalizable way so that the RTE components can be tested generically in extensions.js.

We can probably follow a similar pattern for the tests here and add things ilke richTextEditor.highlightText(...), though we might need to modify richTextEditor.addRteComponent() to let it know to retrieve certain elements from the highlighted value. Probably worth discussing the changes to that API here before implementing it -- I'm not sure what's best offhand, it needs a bit of analysis.